Formula/metacall.rb

```diff
@@ -0,0 +1,105 @@
+class Metacall < Formula
+  desc "Ultimate polyglot programming experience"
+  homepage "https://metacall.io"
+  url "https://github.com/metacall/core/archive/refs/tags/v0.5.27.tar.gz"
+  sha256 "1493afadac893b7b3674ea44dd7caa7fd74989880658ae23003b30d3721bb92d"
+  license "Apache-2.0"
+  head "https://github.com/metacall/core.git", branch: "develop"
+
+  depends_on "cmake" => :build
+  depends_on "node@14"
+  depends_on "openjdk"
+  depends_on "python@3.9"
+  uses_from_macos "ruby"
+
+  def install
+    Dir.mkdir("build")
+    Dir.chdir("build")
+    args = std_cmake_args + %W[
+      -Wno-dev
+      -DCMAKE_BUILD_TYPE=Release
+      -DOPTION_BUILD_SCRIPTS=OFF
+      -DOPTION_FORK_SAFE=OFF
+      -DOPTION_BUILD_TESTS=OFF
+      -DOPTION_BUILD_EXAMPLES=OFF
+      -DOPTION_BUILD_LOADERS_PY=ON
+      -DOPTION_BUILD_LOADERS_NODE=ON
+      -DNodeJS_INSTALL_PREFIX=#{buildpath}
+      -DOPTION_BUILD_LOADERS_JAVA=ON
+      -DOPTION_BUILD_LOADERS_JS=OFF
+      -DOPTION_BUILD_LOADERS_C=OFF
+      -DOPTION_BUILD_LOADERS_COB=OFF
+      -DOPTION_BUILD_LOADERS_CS=OFF
+      -DOPTION_BUILD_LOADERS_RB=ON
+      -DOPTION_BUILD_LOADERS_TS=ON
+      -DOPTION_BUILD_LOADERS_FILE=ON
+      -DOPTION_BUILD_PORTS=ON
+      -DOPTION_BUILD_PORTS_PY=ON
+      -DOPTION_BUILD_PORTS_NODE=ON
+    ]
+    system "cmake", *args, ".."
+    system "cmake", "--build", ".", "--target", "install"
+
+    shebang = "\#!/usr/bin/env bash\n"
+    # debug = "set -euxo pipefail\n"
+
+    metacall_extra = [
+      "LOC=#{prefix}\n",
+      "export LOADER_LIBRARY_PATH=\"#{lib}\"\n",
+      "export SERIAL_LIBRARY_PATH=\"#{lib}\"\n",
+      "export DETOUR_LIBRARY_PATH=\"#{lib}\"\n",
```

Comment:
Why doesn't cmake set this correctly?

Reply:
@SMillerDev Does CMake provide support for installing environment variables? I do not see anything related to that here: https://cmake.org/cmake/help/latest/command/install.html

Reply:
But cmake could set the path where to expect the libraries in the code so you don't need an environment variable to begin with.